Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerPerformanceTrackerTests.swift

```diff
@@ -132,6 +132,7 @@ class SentryUIViewControllerPerformanceTrackerTests: XCTestCase {
             callbackExpectation.fulfill()
         }
         let tracer = try XCTUnwrap(transactionSpan as? SentryTracer)
+        XCTAssertTrue(tracer.shouldIgnore)
```

Comment:
`m`: It seems to me that these test additions don't fully cover the edge case described in GH-4427. I would love to see a test that emulates these edge cases.

Reply:
This addition are meant to check whether `SentryUIViewControllerPerformanceTrackerTests.swift` is properly setting this values.

The test to check whether we create or not the transaction belongs to `SentryTracerTests.swift`